src/test/java/org/folio/rest/impl/DataExportTest.java

```diff
@@ -57,50 +58,40 @@ public DataExportTest() {
     SpringContextUtil.autowireDependencies(this, vertxContext);
   }
 
-
   @Test
-  public void shouldNotCallInventory_whenUploadFileHasOnlyUnderlyingSRS(TestContext context) throws IOException, InterruptedException {
+  public void testExport_UnderlyingSrsOnly(TestContext context) throws IOException {
     Async async = context.async();
-
-    //given
-    ArgumentCaptor<FileDefinition> fileExportDefinitionCaptor = givenCaptureFileExportDefinition();
-    FileDefinition uploadedFileDefinition = givenUploadFile(UUIDS);
-
+    // given
+    String tenantId = okapiConnectionParams.getTenantId();
+    FileDefinition uploadedFileDefinition = uploadFile(UUIDS);
+    ArgumentCaptor<FileDefinition> fileExportDefinitionCaptor = captureFileExportDefinition();
     //when
-    ExportRequest exportRequest = getExportRequest(uploadedFileDefinition);
+    ExportRequest exportRequest = buildExportRequest(uploadedFileDefinition);
     postRequest(JsonObject.mapFrom(exportRequest), EXPORT_URL);
-
+    String jobExecutionId = uploadedFileDefinition.getJobExecutionId();
     // then
-    vertx.setTimer(TIMER_DELAY, handler -> fileDefinitionDao.getById(fileExportDefinitionCaptor.getValue().getId(), okapiConnectionParams.getTenantId())
-      .compose(fileExportDefinitionOptional -> assertCompletedFileDefinitionAndExportedFile(context, fileExportDefinitionOptional))
-      .compose(fileExportDefinition -> jobExecutionDao.getById(fileExportDefinition.getJobExecutionId(), okapiConnectionParams.getTenantId())
-        .compose(jobExecutionOptional -> assertSuccessJobExecution(context, fileExportDefinition, jobExecutionOptional, 2))
-        .compose(optional -> validateExternalCalls(context))
-        .onComplete(succeeded -> async.complete())
-      ));
-  }
-
-  private Future<FileDefinition> validateExternalCalls(TestContext context) {
-    context.assertEquals(1, MockServer.getServerRqRsData(HttpMethod.GET, ExternalPathResolver.SRS).size());
-    context.assertNull(MockServer.getServerRqRsData(HttpMethod.GET, ExternalPathResolver.INSTANCE));
-    context.assertNull(MockServer.getServerRqRsData(HttpMethod.GET, ExternalPathResolver.CONTENT_TERMS));
-    return Future.succeededFuture();
+    vertx.setTimer(TIMER_DELAY, handler -> {
+      jobExecutionDao.getById(jobExecutionId, tenantId).onSuccess(optionalJobExecution -> {
+        JobExecution jobExecution = optionalJobExecution.get();
+        fileDefinitionDao.getById(fileExportDefinitionCaptor.getValue().getId(), tenantId).onSuccess(optionalFileDefinition -> {
+          FileDefinition fileExportDefinition = optionalFileDefinition.get();
+          assertSuccessJobExecution(jobExecution, 2);
```

Comment:
Sorry for being late to the party, but I just observed this, we are doing assertions without the context object, Though when the assertion passes there isn't any problem. If the assertion fails, the test doesn't complete with proper error, but just times out. WE should be careful with this